elastio-stream-oracle-database-backup/backup-oracle-table.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Aha, now I see.  This PR contains multiple `contrib` items, and the previous one had nothing to do with Oracle backup.

Were those other files added by mistake?